content/docs/tasks/security/role-based-access-control/index.md

```diff
@@ -204,7 +215,7 @@ page. But there are errors `"Error fetching product details"` and `"Error fetchi
 are expected because we have not granted "productpage" service to access "details" and "reviews" services. We will fix the errors
 in the following steps.
 
-> There may be delay due to caching on browser and Istio proxy.
+> There may be delay due to caching on browser and policy delayed delivered to Istio proxy.
 
 ### Step 2. allowing "productpage" service to access "details" and "reviews" services
```

Comment:
Why are the steps called out as headers, instead of just using a numbered list? These super long headings don't look good in the table of contents on the rendered web page.

Reply:
I made it a little shorter so it should be able to be rendered in one line in most cases. I think the original idea is to make it more obvious and easier to navigate.